MANIFEST.in

```diff
@@ -75,5 +75,7 @@ recursive-include tests *.json
 recursive-include tests *.mov
 recursive-include tests *.mp4
 recursive-include tests *.py
+recursive-include rfcs *.md
```

Comment:
sorry we should ignore the `rfcs` folder here. That is why this file defines what files are included when we build/publish the package.